htmlbook-xsl/xspec/opf.xspec

```diff
@@ -66,7 +66,7 @@ css_images/pizza.svg
     </x:expect>
   </x:scenario>
 
-  <x:scenario label="When generate.opf is run">
+  <x:scenario label="When generate.opf is run" pending="Setting this scenario as pending because we received the following error when trying to run this scenario: XTDE1425: Cannot find a matching 1-argument function named node-set(). There is no Saxon extension function with the local name node-set. We were using the Saxon JAR file version SAXON HE 9.5.1.2 and the script from https://github.com/xspec/xspec.">

```

Comment:
For these comments, I'd make them shorter. Don't need to say "Setting this scenario as pending because..." That part's implied in the attribute. I'm also not sure you need the whole error. In addition to shortening the message, I'd see if we can format these so that they wrap (to make reviews like this easier).